src/MapzenPlaceConfig.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  MapzenPlaceConfig.swift
+//  Pods
+//
+//  Created by Sarah Lensing on 3/20/17.
+//
+//
+
+import Foundation
+import Pelias
+
+public class MapzenPlaceConfig : NSObject {
```

Comment:
Its against what inherently feels natural to me (its overwhelming to see large files) but if its swift common practice I'll move it into one file.